src/mahoji/lib/abstracted_commands/wintertodtCommand.ts

```diff
@@ -23,7 +23,11 @@ export async function wintertodtCommand(user: MUser, channelID: string) {
 
 	// Up to a 10% boost for 99 WC
 	const wcBoost = (wcLevel + 1) / 10;
-	if (wcBoost > 1) messages.push(`${wcBoost.toFixed(2)}% boost for Woodcutting level`);
+
+	if (wcBoost > 1) {
+		messages.push(`**Boosts: ** ${wcBoost.toFixed(2)}%  for Woodcutting level\n`);
```

Comment:
this string is written weirdly, you're making a space bold - and theres a double space